### ISSUE TITLE
"Dead pixel(s)" at the left bottom of the external screen appears when app is running

### ISSUE THREAD

**Issue by janisgodins** [MonitorControl/MonitorControl#1250]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

A super minor thing, but at first when I noticed it I thought it is hardware issue as it seemed as dead pixel(s) at the left bottom of the Apple Studio Display' corner. It seems it is actually the MonitorControl that puts some black pixel(s) at the corner of the external monitor when running.

### Steps to reproduce

Visible at the left bottom corner of external monitor all the time while the app is running.

### Expected behavior

No black or "dead-like" pixel(s) visible anywhere?

### Anything else?

This is really a minor thing and doesn't bother me much, but could cause some confusion as it looks like as a monitor's hardware issue at first.

### Environment Information (please complete the following information)

```markdown
- macOS version: Ventura 13.1 (RC)
- Mac model: Macbook Pro (14-inch, 2021)
- MonitorControl version: 4.1.0 (7034)
- Monitor(s): Apple Studio Display
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
The app indeed puts a pixel on the screen to trigger screen update when the color table is changing for software dimming. This should not be black however as it's opacity is set to make it invisible. Are you maybe using some kind of color filter or accessibility setting that might make this pixel visible?

Can you post a picture of the problem? Thank you!

**Comment by janisgodins** [comment]
Seems like everything is just regular default settings for system (nothing really customised). Probably hard to see it in a picture, but it just sitting here in the corner.

![IMG_0394](https://user-images.githubusercontent.com/12696435/211904941-31607589-4f29-4f80-90fe-97f46a5637aa.jpg)

**Comment by janisgodins** [comment]
This is from screenshot when zooming in:
<img width="1239" alt="Screenshot 2023-01-11 at 21 59 31" src="https://user-images.githubusercontent.com/12696435/211905714-6365737d-440a-4b09-89d8-798be887655a.png">

**Comment by waydabber** [comment]
Right. That should not be there. :)

**Comment by waydabber** [comment]
Can you check if you have the same issue with [BetterDisplay](https://github.com/waydabber/BetterDisplay)?

**Comment by janisgodins** [comment]
> Can you check if you have the same issue with [BetterDisplay](https://github.com/waydabber/BetterDisplay)?

I tried and seems like this app also have exactly the same black pixel in the corner.

**Comment by waydabber** [comment]
Right, sorry about that. When HDR/XDR upscaling is not enabled in BetterDisplay there is indeed a black dot which is wisible is the screen is zoomed in enough. Fixed that.

**Comment by waydabber** [comment]
Alpha is not set properly on the screen-refresh enforcement pixel for color table control. Similar issue as this: https://github.com/waydabber/BetterDisplay/issues/1365 (as @janisgodins pointed out).

**Comment by Ender-Wang** [comment]
I also run into this problem today, I am migrating from my old m1 mac to the m3 max, I did not see it on my M1, so I started finding solutions what caused it.. Killing processes in the activity monitor and found out it's Moniter Control. I use Bartender and DockMate, those have screen recording access. Not sure what caused that to appear on my M3 mac, since I restored it from a time machine backup.
